src/components/Courses/components/SearchBar/SearchBar.css

```diff
@@ -0,0 +1,7 @@
+.searchBar {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+/* You can add more styles as needed */
```

Comment:
Is this a TODO or a note for me? 😅

Reply:
That was a TODO that I forgot to remove, sorry